src/main/resources/i18n/displayStrings.properties

```diff
@@ -343,6 +343,14 @@ offerbook.warning.tradeLimitNotMatching=Your payment account has been created {0
   You cannot take that offer at the moment. Once your account is older than 2 months this restriction gets removed.
 
 
+offerbook.info.sellAtMarketPrice=You will sell at market price (updated every minute).
+offerbook.info.buyAtMarketPrice=You will buy at market price (updated every minute).
+offerbook.info.sellBelowMarketPrice=You will get {0} less than the current market price (updated every minute).
+offerbook.info.buyAboveMarketPrice=You will pay {0} more than the current market price (updated every minute).
+offerbook.info.sellAboveMarketPrice=You will get {0} more than the current market price (updated every minute).
+offerbook.info.buyBelowMarketPrice=You will pay {0} less than the current market price (updated every minute).
+offerbook.info.buyAtFixedPrice=Your will buy at this fixed price.
+offerbook.info.sellAtFixedPrice=You will sell at this fixed price.
```

Comment:
@ripcurlx, these translations look OK from a native speaker's point of view, but I'm not sure why there is a "sell at / buy at" vs "get / pay" difference. Why not do the following instead?

> offerbook.info.sellBelowMarketPrice=You will **sell at** {0} less than the current market price (updated every minute).
> offerbook.info.buyAboveMarketPrice=You will **buy at** {0} more than the current market price (updated every minute).

I am assuming that the `{0}` placeholder will be replaced with a percent value, btw. e.g. "You will sell at 2% less than the current market price..."

Reply:
Yes, the placeholder is replaced by the absolute percentage distance.
Regarding buy/sell vs. get/pay: I wasn't sure if it might be easier for the user to understand if I use get/pay in some cases instead of buy/sell e.g.

> offerbook.info.sellAboveMarketPrice=You will get {0} more than the current market price (updated every minute).

instead of

> offerbook.info.sellAboveMarketPrice=You will sell at {0} more than the current market price (updated every minute).

But it's just a personal feeling (and I'm no native speaker)

Reply:
I'd favor being consistent. I don't think this is really about native-speaker nuance here, just the general principle that it's better to say the same thing one way versus two ways.

I might make the same point about using the phrase "above the current market price" vs. "more than the current market price", but I don't feel super strongly about it.

Reply:
I'm also a big fan of consistency, but based on some comments on different issues, I have the feeling that it always takes a lot of thinking to understand if above/below in combination with buy/sell is good or bad for me. I think I should start to do more user testing soon to get a better feeling on the biggest pain points within the upcoming release.